### ISSUE TITLE
Checking: Ctrl+C

### ISSUE THREAD

**Issue by gabyx** [rycus86/githooks#117]
@rycus86 : Can you once check Ctrl+C when you do

`git hooks update`
when at the prompt 
`Do you want to install the hooks into existing repositories? [Y/n]`

I cannot SIGINT anymore, it does not react, thats on windows, what do you have on  Linux?

**Comment by rycus86** [comment]
I'll check next time I'm at a computer.
Not sure what could have happened there.

**Comment by gabyx** [comment]
Maybe its just "wincrap" and its bash... 
Maybe our changes with dispatching with `exec` and so forth.... dunno

**Comment by rycus86** [comment]
Yeah, I have some vague memories `exec` can mess things up, but can't remember exactly. Will check when I get a chance.

**Comment by gabyx** [comment]
https://stackoverflow.com/questions/58473685/sending-sigint-to-forked-exec-process-which-runs-script-does-not-kill-it
